### ISSUE TITLE
toc is not defined

### ISSUE THREAD

**Issue by creynders** [assemble/verb#36]
If I try to use `toc` in `.verb.md` it throws an error:

```sh
ReferenceError: toc is not defined
    at eval (eval at template (/usr/local/lib/node_modules/verb/node_modules/lodash/dist/lodash.js:6305:22), <anonymous>:11:26)
    at Function.template (/usr/local/lib/node_modules/verb/node_modules/lodash/dist/lodash.js:6311:16)
    at Object.lodashRender (/usr/local/lib/node_modules/verb/node_modules/template/node_modules/engine-lodash/index.js:77:16)
    at Object.engine.render (/usr/local/lib/node_modules/verb/node_modules/template/node_modules/engine-cache/index.js:178:19)
    at Template.renderAsync (/usr/local/lib/node_modules/verb/node_modules/template/index.js:2019:12)
    at Template.renderBase (/usr/local/lib/node_modules/verb/node_modules/template/index.js:1891:15)
    at Template.renderTemplate (/usr/local/lib/node_modules/verb/node_modules/template/index.js:1969:15)
    at Template.render (/usr/local/lib/node_modules/verb/node_modules/template/index.js:2058:17)
    at DestroyableTransform._transform (/usr/local/lib/node_modules/verb/lib/plugins/render.js:32:12)
    at DestroyableTransform.Transform._read (/usr/local/lib/node_modules/verb/node_modules/through2/node_modules/readable-stream/lib/_stream_transform.js:184:10)
```

I've tried all kinds of variations: `{%= toc() %}`, `{%= toc("*.md") %}`, ... but they all throw the same error.

I'm using the CLI, haven't tried grunt-verb yet.

**Comment by jonschlinkert** [comment]
Good catch, I haven't gotten around to updating the TOC helper - which is a problem if built-in templates are referencing it. I'll do that now.

**Comment by phun-ky** [comment]
+1

**Comment by jonschlinkert** [comment]
I'll get this fixed as soon as I get back from vacation on Thursday

**Comment by jonschlinkert** [comment]
Fixed in 0.7.0. But the following syntax is used now (so we can generate tables of contents at different times than the renderer):

For a single-file TOC, add the following to the place where you want the TOC to render:

```html
<!-- toc -->
```

For a multi-file TOC, add the following to the place where you want the TOC to render:

```html
<!-- toc("*.md") -->
```
